src/components/Input/inputVariables.ts

```diff
@@ -14,6 +14,8 @@ export default (siteVars: any) => {
   vars.inputPadding = `${pxToRem(6)} ${pxToRem(24)} ${pxToRem(6)} ${pxToRem(12)}`
   vars.inputFocusBorderColor = siteVars.brand
   vars.inputFocusBorderRadius = `${pxToRem(3)} ${pxToRem(3)} ${pxToRem(2)} ${pxToRem(2)}`
+  vars.inputMaxWidth = '100%'
```

Comment:
probably, at this point it is not necessary to introduce these customization mechanisms - at least `maxWidth` seems to be an overkill with `width` property being set to the same value. Also, if we would like to introduce `width` variable, we should keep in mind that it is a more specific setting, so it should win `fluent` prop setting for the following case:

```jsx
<Input fluid /> { /* width is 100% */ }
<Input fluid variables={{ width: '50px' }} />  { /* width should be 50px, as 'variables' is more specific setting */ }
```

Reply:
fixing it by reducing the possibility to modify the width of the input tag from variables